From d824bcc17938c60f009ef3dfe0825b9bf03596d4 Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Thu, 9 Nov 2023 17:24:02 +0100
Subject: [PATCH 10/63] ia64: Add futex_wake() syscall

futex_wake() was added after ia64 was removed from the kernel.

Fixes: 9f6c532f59b20 ("futex: Add sys_futex_wake()")
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 6d4fb0ebcb76..99f3ff75e7fc 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -374,3 +374,4 @@
 451	common	cachestat			sys_cachestat
 452	common	fchmodat2			sys_fchmodat2
 453	common	map_shadow_stack		sys_map_shadow_stack
+454	common	futex_wake			sys_futex_wake
-- 
2.25.1

